ledger-api/grpc-definitions/com/daml/ledger/api/v1/event.proto

```diff
@@ -68,11 +68,10 @@ message CreatedEvent {
   // Optional
   Record create_arguments = 4;
```

Comment:
This is probably no longer optional is it? It will be returned in all cases won't it?